architecture/additional_concepts/admission_controllers.adoc

```diff
@@ -34,6 +37,44 @@ you are responsible for updating it to reflect requirements of newer versions of
 the {product-title} master.
 ====
 
```

Comment:
Add an anchor just before the heading, so:

[[admission-controllers-general-admission-rules]]
=== General Admission Rules